Src/VimMac/OsxClipboard.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+static class OsxClipboard
+{
+    static IntPtr nsString = objc_getClass("NSString");
+    static IntPtr nsPasteboard = objc_getClass("NSPasteboard");
+    static IntPtr nsStringPboardType;
+    static IntPtr utfTextType;
+    static IntPtr generalPasteboard;
+    static IntPtr initWithUtf8Register = sel_registerName("initWithUTF8String:");
+    static IntPtr allocRegister = sel_registerName("alloc");
+    static IntPtr setStringRegister = sel_registerName("setString:forType:");
+    static IntPtr stringForTypeRegister = sel_registerName("stringForType:");
+    static IntPtr utf8Register = sel_registerName("UTF8String");
+    static IntPtr generalPasteboardRegister = sel_registerName("generalPasteboard");
+    static IntPtr clearContentsRegister = sel_registerName("clearContents");
+
+    static OsxClipboard()
+    {
+        utfTextType = objc_msgSend(objc_msgSend(nsString, allocRegister), initWithUtf8Register, "public.utf8-plain-text");
```

Comment:
Can we not add a dependency to Xamarin.Mac? This makes the assembly unable to benefit from static registrar codegen.

Reply:
I don't know how to handle the OSX Clipboard any other way

Reply:
Are there no bindings to it in XamMac?

Reply:
I think we can do something slightly better here. We could just get the IntPtr handles via these selectors. Then do `Runtime.GetNSObject<NSPasteBoard> (handle);` for them and try to use as much managed bindings that xammac offers.

For the general pasteboard, there's `NSPasteboard.GeneralPasteboard`.

Reply:
I see that initWithUTF8String is not bound for NSPasteBoard, so we need to use objc_msgSend.

Reply:
OHHH, this one is simple:

```csharp
NSString utfTextType;

utfTextType = new NSString ("public.utf8-plain-text");
```